blackbox/README.md

```diff
@@ -1,6 +1,6 @@
 # Stratis blackbox test run script
 
-The stratis-blackbox-run.sh script builds a set of testing RPM packages of the stratisd and stratis-cli master branches, runs a series of "blackbox" tests against the version of Stratis installed by these packages, and then uninstalls the packages.
+The stratis-blackbox-run.sh script runs a series of "blackbox" tests against thestratisd and stratis-cli packages installed on the system.
```

Comment:
^thestratisd^the stratisd

Reply:
Hmm, that was on column 80, so I didn't even see it.  Fixed via force-push.